lib/telemetry_metrics_statsd/formatter/datadog.ex

```diff
@@ -25,8 +25,15 @@ defmodule TelemetryMetricsStatsd.Formatter.Datadog do
 
   defp format_metric_value(%Metrics.Counter{}, _value), do: "1|c"
 
-  defp format_metric_value(%Metrics.Summary{}, value),
-    do: [format_number(value), "|ms"]
+  defp format_metric_value(
+         %Metrics.Summary{reporter_options: reporter_options},
+         value
+       ) do
+    case Keyword.get(reporter_options, :report_as) do
```

Comment:
I agree it would be nice to have a `datadog` mentioned somewhere in the option name or value. I'll try to come up with something.

Reply:
Let's just go with `report_as: :datadog_distribution`.

Reply:
Done!